src/labels/CustomLabels.labels

```diff
@@ -2145,6 +2145,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>bdiDataImporterTitle</shortDescription>
         <value>NPSP Data Import</value>
     </labels>
+    <labels>
+        <fullName>bdiDeploymentInProgress</fullName>
+        <categories>Data Importer</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>bdiDeploymentInProgress</shortDescription>
+        <value>There is a deployment in progess.</value>
```

Comment:
@psadie this will show when a user tries to enable in a different tab when a deployment is already in progress, right?

If so, please update this message to read:
"Help Text mappings are currently being converted to Custom Metadata."

Is there anything else we should add to that message other than, "Please go put on your patience pants." :) But seriously, should we say something along the lines of, "This page will automatically reload when the conversion is complete."

Reply:
Disregard the last paragraph. Error should read, "Help Text mappings are currently being converted to Custom Metadata."